test/periphery/integration/merkle-campaign/lt/claim/claim.t.sol

```diff
@@ -87,50 +87,69 @@ contract Claim_MerkleLT_Integration_Test is Claim_Integration_Test, MerkleLT_Int
         _;
     }
 
-    function test_Claim_TimestampsInThePast()
+    function test_WhenStreamStartTimeZero()
         external
         whenTotalPercentageOneHundred
         givenCampaignNotExpired
         givenNotClaimed
         givenIncludedInMerkleTree
         whenCalculatedAmountsSumEqualsClaimAmount
     {
-        uint40 streamStartTime = getBlockTimestamp();
         merkleLT = merkleFactory.createMerkleLT({
-            baseParams: defaults.baseParams(users.admin, dai, defaults.EXPIRATION(), defaults.MERKLE_ROOT()),
+            baseParams: defaults.baseParams("lt-start-time-zero", dai),
             lockupTranched: lockupTranched,
             cancelable: defaults.CANCELABLE(),
             transferable: defaults.TRANSFERABLE(),
-            streamStartTime: streamStartTime,
+            streamStartTime: 0,
             tranchesWithPercentages: defaults.tranchesWithPercentages(),
             aggregateAmount: defaults.AGGREGATE_AMOUNT(),
             recipientCount: defaults.RECIPIENT_COUNT()
         });
-        deal({ token: address(dai), to: address(merkleLT), give: defaults.AGGREGATE_AMOUNT() });
-
-        vm.warp(defaults.END_TIME() + 1);
 
-        _test_Claim(streamStartTime);
+        // It should create a stream with block.timestamp as start time.
+        _test_Claim({ streamStartTime: 0, startTime: getBlockTimestamp() });
     }
 
-    function test_Claim()
+    function test_WhenStreamStartTimeNotZero()
         external
-        override
         whenTotalPercentageOneHundred
         givenCampaignNotExpired
         givenNotClaimed
         givenIncludedInMerkleTree
         whenCalculatedAmountsSumEqualsClaimAmount
     {
-        _test_Claim(getBlockTimestamp());
+        merkleLT = merkleFactory.createMerkleLT({
```

Comment:
we don't need to re-deploy this

it is the default `merkleLT` deployed in `MerkleCampaign_Integration_Test`

Reply:
Agree.